celeri/plot_mesh.py

```diff
@@ -0,0 +1,19 @@
+def plot_mesh(meshes, fill_value, ax):
+    import matplotlib.collections
+	for i in range(meshes):
```

Comment:
Indentation should match line above.

Reply:
Edited (saved with black formatting) in a876f1233bb3b0e1b63337fd2e8b9800816d7e6c